music21/mei/base.py

```diff
@@ -2493,6 +2504,52 @@ def chordFromElement(elem, slurBundle=None):
 
     return theChord
 
+def harmFromElement(elem, slurBundle=None):
+    # other tags than <fb> to be added…
+    tagToFunction = {f'{MEI_NS}fb': figuredbassFromElement}
+
+    fb_harmony_tag: tuple = ()
+    
+    # Collect all elements in a measure and go throug extenders
+    # tstamp has to be used as a duration marker between two elements
+
+    for subElement in _processEmbeddedElements(elem.findall('*'),
+                                               tagToFunction,
+                                               elem.tag, slurBundle):
+        subElement.tstamp = float(elem.get('tstamp'))
+        subElement.offset = subElement.tstamp - 1
+        fb_harmony_tag = (subElement.tstamp - 1, subElement)
+
+    return fb_harmony_tag
+
+def figuredbassFromElement(elem, slurBundle=None):
+    if elem.get(_XMLID):
+        fb_id = elem.get(_XMLID)
+    fb_notation = ''
+    dauer: float = 0
+    # loop through all child elements and collect <f> tags
+    for subElement in elem.findall('*'):
```

Comment:
You don't need to loop through all of the elements -- you can just do an xquery to find just the `<f>` elements and return a list of those.

Reply:
Do you mean something like `elem.findall('f'{MEI_NS}f'')`? Then the first if statement is not necessary. I think I did this to be able to grab other harmony tags here in the future.